packages/playground/assets/__tests__/assets.spec.ts

```diff
@@ -105,6 +105,27 @@ describe('css url() references', () => {
     })
   })
 
+  test('image-set with var', async () => {
+    const imageSet = await getBg('.css-image-set-with-var')
+    imageSet.split(', ').forEach((s) => {
+      expect(s).toMatch(assetMatch)
+    })
+  })
+
+  test('image-set with mix', async () => {
+    const imageSet = await getBg('.css-image-set-mix-url-var')
+    imageSet.split(', ').forEach((s) => {
+      expect(s).toMatch(assetMatch)
+    })
+  })
+
+  // test('image-set with multiple descriptor', async () => {
+  //   const imageSet = await getBg('.css-image-set-multiple-descriptor')
+  //   imageSet.split(', ').forEach((s) => {
+  //     expect(s).toMatch(assetMatch)
+  //   })
+  // })
```

Comment:
@Bigfish8 is there a reason for this test to be commented out?

Reply:
@pakaf because browser do not support this case now but mentioned in [spec](https://drafts.csswg.org/css-images-4/#funcdef-image-set-type).

> \<image-set-option\> = [ \<image\> | \<string\> ]  [ \<resolution\> | | type(\<string\>)]
The `||` here meaning `1x` and `type("image/jpeg")` may both exists.

Reply:
Would you add a comment if we are going to leave this test case commented out? If not, we could remove it and create an issue for later.

Reply:
Of course!
Add the comment.